src/nn/wasm/WorkerIPC.js

```diff
@@ -0,0 +1,31 @@
+import PreparedModelWorker from 'worker-loader?inline!./PreparedModelWorker';
+
+export function getPreparedModelWorker() {
+  let _worker = new PreparedModelWorker();
+  let _handlers = {};
+  let _handlerCounter = 0;
+  _worker.onmessage = function (e) {
+    const [id, fn, ret, error] = e.data;
+    const fnId = fn + id;
+    if (!error) {
+      const resolveHandler = _handlers[fnId][0];
+      resolveHandler(ret);
+    } else {
+      const err = new Error();
+      [err.message, err.stack] = ret;
+      const rejectHandler = _handlers[fnId][0];
```

Comment:
`const rejectHandler = _handlers[fnId][1];`?

Reply:
I think I should document on this and other helper functions later.
The handler is registered in this line [`_handlers[fn + id] = [resolve, reject];`](https://github.com/intel/webml-polyfill/pull/521/files/960b02dfeb5b5504eba93588d8ba93c9809c5264#diff-c4df521adfc067a4550fe853b7f72430R27)

the structure of `_handlers`:
```js
_handlers = {
  execute135: [resolve, reject],
  deleteAll136: [resolve, reject],
}
```

if we call 
```js
await worker.dispatch('getNNOpsInstance');
```
`dispatch` will register a new entry to the `_handlers`, then returns a Promise that can be resolved by the `_handlers['getNNOpsInstance137'][0]` or rejected by the `_handlers['getNNOpsInstance137'][1]`.
```js
_handlers = {
  execute135: [resolve, reject],
  deleteAll136: [resolve, reject],
  getNNOpsInstance137: [resolve, reject],
}
```


Once the main thread receives a message from the worker, it will resolve/reject the corresponding Promise.
```js
  // in main thread
  _worker.onmessage = function (e) {
    const [id, fn, ret, error] = e.data;
    const fnId = fn + id;
    if (!error) {
      const resolveHandler = _handlers[fnId][0];
      resolveHandler(ret);
    }
...
```

Reply:
Sorry, it's a typo. Fixed